dlt/pipeline/pipeline.py

```diff
@@ -392,6 +390,12 @@ def drop(self, pipeline_name: str = None) -> "Pipeline":
             self.config,
             self.runtime_config,
         )
+        new_pipeline.activate()
```

Comment:
I suppose there isn't  a way to replace this pipeline with the new one so that this is transparent to the user? I would probably expect to be able to do:
p.run(...)
p.drop()
p.run(...)

Reply:
I completely reworked this part. And now it works like you say :)